components/backup-stream/src/router.rs

```diff
@@ -1785,11 +1800,13 @@ impl MetadataInfo {
             .map_err(|err| Error::Other(box_err!("failed to marshal proto: {}", err)))
     }
 
-    fn path_to_meta(&self) -> String {
+    fn path_to_meta(&self, min_begin_ts: u64, flush_ts: u64) -> String {
         format!(
-            "v1/backupmeta/{}-{}.meta",
-            self.min_resolved_ts.unwrap_or_default(),
-            uuid::Uuid::new_v4()
+            "v1/backupmeta/{:X}-{:X}-{:X}-{:X}.meta",
```

Comment:
```suggestion
            "v1/backupmeta/{:016X}-{:016X}-{:016X}-{:016X}.meta",
```

Pad zeros to avoid unaligned file names.